include/bm/bm_sim/calculations.h

```diff
@@ -387,6 +387,32 @@ class CustomCrcMgr {
                                           const crc_config_t &config);
 };
 
+enum class RssErrorCode {
+  SUCCESS = 0,
+  INVALID_CALCULATION_NAME,
+  WRONG_TYPE_CALCULATION,
+  INVALID_KEY,
+};
+
+namespace detail {
+
+using rss_key_t = std::vector<uint8_t>;
+
+std::ostream &operator<<(std::ostream &out, const rss_key_t &rss_key);
+
+}  // namespace detail
+
+class RssMgr {
```

Comment:
is this going to be used for anything else? because as a standalone object, I find the name a bit confusing. If it will be specific to the Toeplitz hash function, then the name `ToeplitzRssMgr` would be better. Same for other names (e.g., `set_rss_key` -> `set_toeplitz_rss_key`).

Reply:
Done.